src/common.cpp

```diff
@@ -903,11 +903,6 @@ int common_create_window(HINSTANCE hInstance, struct game_obj* game_object)
 				if (more_debug)
 				{
 					replace_function(common_externals.debug_print2, external_debug_print2);
-
-					if (ff8)
-					{
-						patch_code_dword(ff8_externals.outputdebugstringa, DWORD(external_debug_print));
```

Comment:
What is the reasoning of this removal other than fixing the bug? Shall we maybe hook the sub or do we lose any functionality here?

Reply:
I don't know how to fix the bug, I though replacing the dynamic address of OutputDebugStringA was a good idea, it works for the first call, and then the program is in a undefined state

Reply:
Ok, let me have a look at this after work and I'll let you know if I find a better way to fix it other than removing the patch itself :) Thank you

Reply:
You can quickly reproduce the bug/crash by launching the game and quickly after change to another application window

Reply:
Allright, tested but I'm not able to find any possible solution atm. Guess we'll merge the patch as it is, to eventually unpack this eventually another day. Thanks for catching this bug :)